paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -87,25 +83,6 @@ internal class PaymentOptionsActivityTest {
 
     private val eventReporter = mock<EventReporter>()
     private lateinit var injector: NonFallbackInjector
-    private val lpmRepository = mock<LpmRepository>().apply {
-        whenever(fromCode(any())).thenReturn(
```

Comment:
Removing this mock and replacing it with the real object so that tests involving payment methods other than cards work correctly.